### PR TITLE
Kat Furniture! New sofas, chess pieces, nine new posters and shower curtain functionality!

### PR DESCRIPTION
**Pull request by KathrinBailey**
## About The Pull Request
Adds the following features:
• New sofas with armrest and buckle functionality.
• Buildable green and purple comfy chairs, to fit in with green and purple carpets and tables.
• Shower curtains now block sight only once closed, and let you see through them once open!
• Shower curtains can be crafted with ten plastic sheets.
• Nine new posters!
• Chess pieces!

## Why It's Good For The Game
Adds new assets. The sofas are already used on other sofas and add variety for both mappers and players, as they can craft them.
Comfy chair colours are a bit bland. Green and purple chairs seem like okay options for new colours without cluttering the list too much.
Shower curtain functionality gives a lot of freedom for mappers. It also opens up a ton of roleplay potential. Changing booths for pools? Check!
Chess pieces. This is not intended to be a round-start feature. You are supposed to make a chess board to play. This stops chess being played every shift until it gets bland, you have to put a bit of effort into it.
Nine new posters that are all in-universe and can be used in roleplay. I did a blind tent and asked people to point out what posters are new and asked them if any don't fit in (no one said they don't). The results are at the bottom of this PR. I accidentally put 21 and 22 twice. Oops.

## What is required after this PR?
Anyone doing a bugfix on BoxStation should delete the tinted windows under the curtains. These windows were added so that the shower curtains blocked sight at all times. It was either that, or it allows sight at all times.
If you do a bugfix on BoxStation, please add the HoochMaster (bar chem-master) to the bar. I forgot to re-add it back ages ago and it's my fault. It was never supposed to be removed, hence it's a bug.

## Changelog
:cl:
add: Nine new posters!
add: Shower curtains can be crafted.
add: New sofas!
add: Green and purple comfy chairs to the crafting menu to fit green and purple carpets.
fix: Shower curtains now let you see through them once open, and don't once closed.
/:cl:

## Additional Credits
• Whoever made the original sofa sprites. They were ported from Hyper, who ported them from elsewhere. I edited the sprites a bit and added arm-rests.
• Ghommie for suggesting improvements to the original Chess PR.

## Images
image
**The new posters are:
• NTAPDA 600
• NTAPDA 800
• Hydro
• SCUM
• NT Storm
• NT Storm, but the other version
• Medical (Green +)
• Syndicate (White S but snake head)
• Fashion!**

## Blind Test
image
image
image
**Answers:**
_6, 8, 9, 10, 11, 13, 15, 24 and 26._

### CODE REVIEW

**Comment by Detective-Google** [comment]
I'm pretty sure those sofa sprites come from Eris, which I've said a few times. If I was at my computer I'd go find it on their code and show you, but generally it's a dick move not to credit the sprite authors when porting sprites.

**Comment by KathrinBailey** [comment]
> I'm pretty sure those sofa sprites come from Eris, which I've said a few times. If I was at my computer I'd go find it on their code and show you, but generally it's a dick move not to credit the sprite authors when porting sprites.

And afterwards, I checked maps to look around for CEV Eris and couldn't find any of the sofas there to confirm it.
I actually gave credit for the sofas, twice, but it seems like the full credit is in the backup files I made and I didn't copy and paste it over. I'm re-adding the credits that were unintentionally missed off.

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -414,6 +414,16 @@
...
 
+/obj/structure/sign/poster/contraband/scum
+	name = "Security are Scum"
+	desc = "Anti-security propaganda. Features a human NanoTrasen security officer being shot in the head, with the words 'Scum' and a short inciteful manifesto. Used to anger security."
+	icon_state = "poster46"
+
+/obj/structure/sign/poster/contraband/scum
+	name = "Security are Scum"
+	desc = "Anti-security propaganda. Features a human NanoTrasen security officer being shot in the head, with the words 'Scum' and a short inciteful manifesto. Used to anger security."
+	icon_state = "poster47"
```
Comment (Ghommie) [inline code/game/objects/effects/contraband.dm]: Same description, same name, same path, just different icon states.
I believe the second is pertinent to something else and not security though.

File: code/game/objects/structures/chess.dm [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+/obj/structure/chess/wrench_act(mob/user, obj/item/tool)
+	to_chat(user, "<span class='notice'>You take apart the chess piece.</span>")
+	var/obj/item/stack/sheet/metal/M = new (drop_location(), 2)
+	M.add_fingerprint(user)
+	tool.play_tool_sound(src)
+	qdel(src)
+	return TRUE
+
+/obj/structure/chess/WhitePawn
```
Comment (Ghommie) [inline code/game/objects/structures/chess.dm]: keep the paths lowercase.

File: code/game/objects/structures/watercloset.dm [hunk 3]
```diff
@@ -633,12 +634,14 @@
 		layer = WALL_OBJ_LAYER
 		density = TRUE
 		open = FALSE
+		opacity = 1
```
Comment (Ghommie) [inline code/game/objects/structures/watercloset.dm]: TRUE

File: code/game/objects/structures/watercloset.dm [hunk 4]
```diff
@@ -633,12 +634,14 @@
...
 
 	else
 		icon_state = "open"
 		layer = SIGN_LAYER
 		density = FALSE
 		open = TRUE
+		opacity = 0
```
Comment (Ghommie) [inline code/game/objects/structures/watercloset.dm]: FALSE

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/beds_chairs/sofa.dm [hunk 5]
```diff
@@ -0,0 +1,39 @@
+// Credit for the sprites goes to CEV Eris. The sprites were taken from Hyper Station and modified to fit with armrests which were also added.
+
+/obj/structure/chair/corp_sofa
```
Comment (Ghommie) [inline code/game/objects/structures/beds_chairs/sofa.dm]: Move normal sofas into this file amd make these ones a subtype of them.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Just noticed it was a new file.

**Comment by KathrinBailey** [comment]
Change requests are done.
Armrest spawning on wrong sofa dir also fixed, armrests tested and working.

File: code/game/objects/structures/beds_chairs/sofa.dm [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+/obj/structure/chair/sofa/corner/handle_layer() //only the armrest/back of this chair should cover the mob.
+	return
+
+// Credit for the sprites goes to CEV Eris. The sprites were taken from Hyper Station and modified to fit with armrests which were also added.
+
+/obj/structure/chair/sofa/corp
+	name = "sofa"
+	desc = "Soft, cushy and cozy. These sofas reek of bland faceless corporatism, but they aren't old and ratty at least."
+	icon_state = "corp_sofamiddle"
+	icon = 'icons/obj/sofa.dmi'
```
Comment (Ghommie) [inline code/game/objects/structures/beds_chairs/sofa.dm]: remove.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**